.storybook/storyOrder.js

```diff
@@ -1,20 +0,0 @@
-const README = ["Welcome", "Getting Started", "Contributing"];
```

Comment:
This readme section is missing. Will it show up without it?

Reply:
Yeah, it looks like storybook 7 automatically puts docs first so there's no need to specify these anymore

Reply:
That said, keeping `Welcome` first is a good idea. I added a commit that brings these back